client/app/components/form/MultiStepAccordion.tsx

```diff
@@ -0,0 +1,68 @@
+"use client";
+
+import { useState } from "react";
+import Accordion from "app/components/accordion/Accordion";
+import FormBase from "./FormBase";
+
+interface Props {
+  schema: any;
+  uiSchema: any;
+  formData: any;
```

Comment:
Thanks, this was from early development and I forgot to go back. Added types [here](https://github.com/bcgov/cas-registration/pull/796/commits/674321de436ff03654a7941af05213624c26a043) except for the MultiStepAccordion `formData` type I wasn't sure of what to add since it can take any type of formData.